### PR TITLE
minor enhancements to hammer_cli(_foreman) packaging (DEB)

### PR DESCRIPTION
**Pull request by mmoll**
- set license
- include doc directory into docs
- use config file in the gem (not yet in _foreman gem)
- scratchbuilds:
 - http://ci.theforeman.org/job/packaging_build_deb_dependency/617/
 - http://ci.theforeman.org/job/packaging_build_deb_dependency/618/

### CODE REVIEW

File: dependencies/precise/hammer_cli_foreman/copyright [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
-Copyright: <years> <put author's name and email here>
-           <years> <likewise for another author>
-License: GPL-2+ (FIXME)
- This program is free software; you can redistribute it
- and/or modify it under the terms of the GNU General Public
- License as published by the Free Software Foundation; either
- version 2 of the License, or (at your option) any later
- version.
+Copyright: 2012 Brian Gupta <brian.gupta@brandorr.com>
+           2013-2014 RedHat Inc. <foreman-dev@googlegroups.com>
```
Comment (domcleal) [inline dependencies/precise/hammer_cli_foreman/copyright]: I'd maybe say "Foreman developers" here, as it isn't exclusively Red Hat (extra space)

File: dependencies/precise/hammer_cli_foreman/copyright [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
- License version 2 can be found in the file
- `/usr/share/common-licenses/GPL-2'.
+ On Debian systems, the full text of the GNU General Public License
+ version 3 can be found in the file `/usr/share/common-licenses/GPL-3'.
 
 Files: debian/*
-Copyright: 2013  <>
-License:
- [LICENSE TEXT]
+Copyright: 2013 Tomáš Strachota <tstrachota@redhat.com>
```
Comment (domcleal) [inline dependencies/precise/hammer_cli_foreman/copyright]: either Foreman devs again perhaps, or @tstrachota may be able to confirm, but I expect it's copyright Red Hat if he did it on company time

**Comment by domcleal** [comment]
:+1: aside from nitpicking

**Comment by domcleal** [comment]
Oh, I didn't fully read that.. depends on a new release of ..foreman with https://github.com/theforeman/hammer-cli-foreman/pull/163

**Comment by mmoll** [comment]
The "new-style" config file handling is implemented only in hammer_cli, not hammer_cli_foreman, so this could be merged as-is.

The copyrights were ACKed by @mbacovsky via IRC, but pending comments from him or @tstrachota I'll change these to "Red Hat Inc." and "Foreman developers", as there can be multiple entries and that should be safe.

**Comment by domcleal** [comment]
:+1:

File: dependencies/precise/hammer_cli_foreman/copyright (hunk 2 above)
Comment (tstrachota) [inline dependencies/precise/hammer_cli_foreman/copyright]: Company time, but I guess Foreman devs are ok.

File: dependencies/precise/hammer_cli_foreman/copyright (hunk 1 above)
Comment (tstrachota) [inline dependencies/precise/hammer_cli_foreman/copyright]: +1

**Comment by tstrachota** [comment]
Nice cleanup. Thanks @mmoll ! 
:+1: when @domcleal's comments are fixed. It's not blocked by https://github.com/theforeman/hammer-cli-foreman/pull/163 but it's merged now so you can work it in if you want.

**Comment by mmoll** [comment]
Should be good to go now :)

**Comment by tstrachota** [comment]
Ack @mmoll @domcleal

**Comment by domcleal** [comment]
Thanks @mmoll, merged as 1879d32, e131aa7.

**Pull request closed without merging**